From 9295d78a419ea0504368b42228f69ccab1171c79 Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Sun, 27 May 2018 14:44:20 +0200
Subject: [PATCH 1/2] klte-common: Inherit shims from msm8974-common

* And split shims by category while we're at it

Change-Id: I1200f025edf8ba10265c806acbda35d023cdd405
---
 BoardConfigCommon.mk | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/BoardConfigCommon.mk b/BoardConfigCommon.mk
index 90e65fb..6d3eb1c 100644
--- a/BoardConfigCommon.mk
+++ b/BoardConfigCommon.mk
@@ -35,6 +35,7 @@ TARGET_BOOTLOADER_BOARD_NAME := MSM8974
 
 # Camera
 USE_DEVICE_SPECIFIC_CAMERA := true
+TARGET_LD_SHIM_LIBS += /system/vendor/lib/hw/camera.vendor.msm8974.so|libshim_camera.so
 
 # Extended Filesystem Support
 TARGET_EXFAT_DRIVER := sdfat
@@ -56,7 +57,6 @@ TARGET_KERNEL_SOURCE := kernel/samsung/msm8974
 
 # Legacy BLOB Support
 TARGET_NEEDS_PLATFORM_TEXT_RELOCATIONS := true
-TARGET_LD_SHIM_LIBS := /system/vendor/lib/hw/camera.vendor.msm8974.so|libshim_camera.so:/system/lib/libcutils.so|libshim_cutils_atomic.so
 
 # Partitions
 BOARD_BOOTIMAGE_PARTITION_SIZE := 13631488
@@ -79,6 +79,7 @@ TARGET_SYSTEM_PROP += $(COMMON_PATH)/system.prop
 # Radio
 BOARD_PROVIDES_LIBRIL := true
 TARGET_RIL_VARIANT := caf
+TARGET_LD_SHIM_LIBS += /system/lib/libcutils.so|libshim_cutils_atomic.so
 
 # Recovery
 BOARD_CUSTOM_RECOVERY_KEYMAPPING := ../../$(COMMON_PATH)/recovery/recovery_keys.c
-- 
2.17.0

